stellar/README.md

```diff
@@ -10,28 +10,31 @@ brew install libsodium
 npm ci
 ```
 
-Install Soroban CLI
+Install Stellar Soroban CLI
 
 ```bash
-cargo install --locked soroban-cli --features opt
+cargo install --locked stellar-cli --features opt
```

Comment:
interesting, is the `opt` feature still relevant/needed?

Reply:
as per [the stellar setup doc](https://developers.stellar.org/docs/build/smart-contracts/getting-started/setup), Yes, it's still needed if we want to use `cargo`